src/rdkafka_broker.c

```diff
@@ -624,7 +631,16 @@ static int rd_kafka_broker_bufq_timeout_scan (rd_kafka_broker_t *rkb,
 
 		rd_kafka_bufq_deq(rkbq, rkbuf);
 
-		if (is_waitresp_q && rkbuf->rkbuf_flags & RD_KAFKA_OP_F_BLOCKING
+                if (now && cnt < log_first_n)
+                        rd_rkb_log(rkb, LOG_NOTICE, "REQTMOUT",
```

Comment:
is it worth logging an additional message saying there are or summarizing the other log messages that are suppressed?

Reply:
There is a "%i request(s) timed out .."  log from the caller